translations/strings.yaml

```diff
@@ -677,6 +677,17 @@ tutorialPopup:
   tutorialLeaveSubtitle: Looks like you haven’t finished the tutorial quite yet. Would you like to continue and finish now or quit and come back later?
   tutorialLeavePrimaryButtonLabel: Resume tutorial
 
+navBar:
+  homeTab:
+    value: Home
+    comment: Label for the tab on the nav bar that takes the user to the home screen
```

Comment:
übernit: `for the tab on the nav bar` is incredibly hard to parse for me (because of the amount of short words), can we go with `for the tab on the navigation bar`?